angrmanagement/ui/widgets/qinstruction.py

```diff
@@ -8,13 +8,18 @@
 from .qoperand import QOperand
 from ...utils import should_display_string_label, get_string_for_display, get_comment_for_display
 
+import logging
+l = logging.getLogger(name=__name__)
+l.setLevel('INFO')
```

Comment:
Do not explicitly call `logger.setLevel()` in committed code. Also, `import logging` should go to the top of this file.